client/src/containers/navbar/index.js

```diff
@@ -1,15 +1,15 @@
 import { bindActionCreators } from 'redux'
 import { connect } from 'react-redux'
-import { authorizeUser, fetchUser, logoutUser } from '../../actions'
+import { authorizeUser, logoutUser, fetchUser } from '../../actions'
 
 import Navbar from '../../components/navbar'
 
 const mapDispatchToProps = dispatch =>
   bindActionCreators(
     {
       authorizeUser,
-      fetchUser,
-      logoutUser
+      logoutUser,
+      fetchUser
```

Comment:
Did you do this or was it a result of a rebase?

Reply:
Result of a merge.